x/multi-staking/types/key.go

```diff
@@ -31,3 +36,28 @@ var (
 	// mem store key
 	CompletedDelegationsPrefix = []byte{0x00}
 )
+
+// GetBondTokenWeightKeyKey returns a key for an index containing the bond token weight
+func GetBondTokenWeightKey(tokenDenom string) []byte {
+	return append(BondTokenWeightKey, []byte(tokenDenom)...)
+}
+
+// GetValidatorBondDenomKey returns a key for an index containing the bond denom of a validator
+func GetValidatorBondDenomKey(operatorAddr sdk.ValAddress) []byte {
+	return append(ValidatorBondDenomKey, address.MustLengthPrefix(operatorAddr)...)
+}
+
+// GetIntermediaryAccountDelegatorKey returns a key for an index containing the delegator of an intermediary account
+func GetIntermediaryAccountDelegatorKey(intermediaryAccount sdk.AccAddress) []byte {
+	return append(IntermediaryAccountDelegator, intermediaryAccount...)
+}
+
+func GetDVPairSDKBondTokensKey(delAddr sdk.AccAddress, valAddr sdk.ValAddress) []byte {
+	DVPair := append(delAddr, address.MustLengthPrefix(valAddr)...)
+	return append(DVPairSDKBondTokens, DVPair...)
+}
+
+func GetDVPairBondTokensKey(delAddr sdk.AccAddress, valAddr sdk.ValAddress) []byte {
+	DVPair := append(delAddr, address.MustLengthPrefix(valAddr)...)
```

Comment:
same about `MustLengthPrefix`

Reply:
I think it's okay to leave a panic there